pendulum:
    start_cartx: 0
    start_angle: 1
    track_length: 1.5
    gui_update_frequency: 20
    motor:
        port: "/dev/pololu_motor_controller"
    potentiometer:
        high: 948
        low: 70
    pid:
        update_frequency: 10
        angle:
            p: 0
            i: 0.0
            d: 0
        cart:
            p: 70
            i: 0.1
            d: 50

